Modules/Project/Entities/Project.php

```diff
@@ -99,7 +99,17 @@ public function getExpectedHoursInMonthAttribute($startDate = null, $endDate = n
     {
         $startDate = $startDate ?? $this->client->month_start_date;
         $endDate = $endDate ?? $this->client->month_end_date;
-        $daysInMonth = count($this->getWorkingDaysList($startDate, $endDate));
+        $currentdate = today(config('constants.timezone.indian'));
+        $currentmonth = today(config('constants.timezone.indian'));
+        $currentmonth = $currentmonth->format('m');
+        $currentyear = today(config('constants.timezone.indian'));
+        $currentyear = $currentyear->format('y');
+        $usermonth = $startDate->format('m');
+        $useryear = $startDate->format('y');
+        if (($currentmonth == $usermonth) && ($currentyear == $useryear)) {
+            $endDate = $currentdate;
+        }
+        $daysInMonth = count($this->getWorkingDaysList($startDate, $endDate)) - 1;
```

Comment:
You don't need to pass `config('constants.timezone.indian')` as the default the timezone is set in .env and we have set it to IST in the application

```suggestion
        $currentMonth = today()->format('m');
        $currentYear = today()->format('y');
        $projectMonth = $startDate->format('m');
        $projectYear = $startDate->format('y');
        if (($currentMonth == $projectMonth) && ($currentYear == $projectYear)) {
            $endDate = today();
        }
        $daysInMonth = count($this->getWorkingDaysList($startDate, $endDate)) - 1;
```